sdks/java/core/src/test/java/org/apache/beam/sdk/util/RetryHttpRequestInitializerTest.java

```diff
@@ -50,6 +50,7 @@
 import com.google.api.services.storage.Storage;
 import com.google.common.collect.ImmutableList;
 
+import org.apache.beam.sdk.options.PipelineOptionsFactory;
```

Comment:
[ERROR] /home/jenkins/jenkins-slave/workspace/beam_PreCommit_MavenVerify/sdks/java/core/src/test/java/org/apache/beam/sdk/util/RetryHttpRequestInitializerTest.java:53: Import org.apache.beam.sdk.options.PipelineOptionsFactory appears after other imports that it should precede [ImportOrder]